plugins/jobtype/src/azkaban/jobtype/HadoopHiveJob.java

```diff
@@ -22,12 +22,15 @@
 import java.io.IOException;
```

Comment:
Reportal's pig and hive seem to use a completely different code base.
We can do it, but I don't think should be in this pull request.